src/scripts_lut.c

```diff
@@ -0,0 +1,1351 @@
+  0x99, 0x00, 0x1e, 0x0e, 0x00, 0x9f, 
+};
+const unsigned char WriteMem8_0[20] = {
+  0x91, 0x00, 0x91, 0x01, 0xad, 0x01, 0x99, 0x03, 0x1e, 0x06, 0x00, 0x03, 0x92, 0x00, 0x01, 0x00, 0x00, 0x00, 0xae, 0x5a, 
+};
+const unsigned char WriteMem16_0[23] = {
+  0x91, 0x00, 0x91, 0x01, 0x67, 0x01, 0x01, 0xad, 0x01, 0x9a, 0x03, 0x1e, 0x07, 0x00, 0x03, 0x92, 0x00, 0x02, 0x00, 0x00, 0x00, 0xae, 0x5a, 
+};
+const unsigned char ReadMem8_0[19] = {
+  0x91, 0x00, 0x91, 0x01, 0x95, 0xad, 0x01, 0x1e, 0x03, 0x00, 0x9f, 0x92, 0x00, 0x01, 0x00, 0x00, 0x00, 0xae, 0x5a, 
+};
+const unsigned char ReadMem16_0[22] = {
+  0x91, 0x00, 0x91, 0x01, 0x95, 0x67, 0x01, 0x01, 0xad, 0x01, 0x1e, 0x04, 0x00, 0x9d, 0x92, 0x00, 0x02, 0x00, 0x00, 0x00, 0xae, 0x5a, 
+};
+const unsigned char ReadSIB_0[7] = {
+  0x95, 0x9b, 0x00, 0x02, 0x1e, 0x12, 0x00, 
+};
+const unsigned char HoldInReset_0[38] = {
+  0x9b, 0x00, 0x08, 0x9b, 0x01, 0x00, 0x1e, 0x01, 0x00, 0x01, 0x9b, 0x00, 0x08, 0x9b, 0x01, 0x59, 0x1e, 0x0f, 0x00, 0x01, 0x9b, 0x01, 0x0b, 0xa2, 0x1e, 0x0e, 0x01, 0xa5, 0x20, 0x00, 0x00, 0x00, 0x20, 0x00, 0x00, 0x00, 0x0a, 0x00, 
+};
+const unsigned char ReleaseFromReset_0[40] = {
+  0x9b, 0x00, 0x08, 0x9b, 0x01, 0x00, 0x1e, 0x01, 0x00, 0x01, 0x9b, 0x00, 0x08, 0x9b, 0x01, 0x00, 0x1e, 0x0f, 0x00, 0x01, 0x9b, 0x01, 0x0b, 0xa2, 0x1e, 0x0e, 0x01, 0xa5, 0x20, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x0a, 0x00, 0x1e, 0x02, 
+};
+
+
+
+struct avr_script_lut avr_scripts = {
+  .EnterProgMode = EnterProgMode_0,
+  .EnterProgModeHvSp = EnterProgModeHvSp_0,
+  .EnterProgModeHvSpRst = EnterProgModeHvSpRst_0,
+  .EnterProgModeHvUpt = EnterProgModeHvUpt_0,
+  .ExitProgMode = ExitProgMode_0,
+  .SetSpeed = SetSpeed_0,
+  .GetDeviceID = NULL,
+  .EraseChip = EraseChip_0,
+  .ReadProgmem = NULL,
+  .WriteDataEEmem = NULL,
+  .ReadDataEEmem = NULL,
+  .WriteCSreg = WriteCSreg_0,
+  .ReadCSreg = ReadCSreg_0,
+  .WriteMem8 = WriteMem8_0,
+  .WriteMem16 = WriteMem16_0,
+  .ReadMem8 = ReadMem8_0,
+  .ReadMem16 = ReadMem16_0,
+  .ReadSIB = ReadSIB_0,
+  .HoldInReset = HoldInReset_0,
+  .ReleaseFromReset = ReleaseFromReset_0,
+  .EnterProgMode_len = sizeof(EnterProgMode_0),
+  .EnterProgModeHvSp_len = sizeof(EnterProgModeHvSp_0),
+  .EnterProgModeHvSpRst_len = sizeof(EnterProgModeHvSpRst_0),
+  .EnterProgModeHvUpt_len = sizeof(EnterProgModeHvUpt_0),
+  .ExitProgMode_len = sizeof(ExitProgMode_0),
+  .SetSpeed_len = sizeof(SetSpeed_0),
+  .GetDeviceID_len = 0,
+  .EraseChip_len = sizeof(EraseChip_0),
+  .ReadProgmem_len = 0,
+  .WriteDataEEmem_len = 0,
+  .ReadDataEEmem_len = 0,
+  .WriteCSreg_len = sizeof(WriteCSreg_0),
+  .ReadCSreg_len = sizeof(ReadCSreg_0),
+  .WriteMem8_len = sizeof(WriteMem8_0),
+  .WriteMem16_len = sizeof(WriteMem16_0),
+  .ReadMem8_len = sizeof(ReadMem8_0),
+  .ReadMem16_len = sizeof(ReadMem16_0),
+  .ReadSIB_len = sizeof(ReadSIB_0),
+  .HoldInReset_len = sizeof(HoldInReset_0),
+  .ReleaseFromReset_len = sizeof(ReleaseFromReset_0),
+};
+
+
+char* chip_lut[] = {
```

Comment:
There are a few "new" parts here like `AVR16DV14`, `AVR64EC48` and `AVR32SD32`. These must be typos, right?

Reply:
Never mind. Found traces of these parts in scripts.xml

Reply:
Nope, no typos, They are either in Developement (EC, SD) or older, non released ones. At least I think so. 
The scripts_lut files are automatically generated, it's directly pulled from the description file called "scripts.xml".
The chip designers are probably using the Debugger already with the FPGAs for developement, so they had to add it in advance. But this isn't a big deal, as, except for some memory ranges, not much is getting disclosed that way anyway.
I could add a black-list routine, but decided against it, as this would make it easier providing support in the future.